ma/api/media.py

```diff
@@ -164,5 +164,9 @@ def create(self, product_id, rel_filepath, label_text, for_types,
 
         return upload_rel_filepath
 
+    def update(self, product_id, rel_filepath, label_text, for_types,
+               position=0):
+        pass
```

Comment:
Darn... media.py should not have been included. Part of a different branch with non finished functionality. I thought I stashed that.